web/src/components/DandisetsPage.vue

```diff
@@ -39,33 +40,61 @@
           </v-list-item>
         </v-list>
       </v-menu>
-      <div class="mx-6">
-        Sort By:
-      </div>
-      <v-chip-group
-        :model-value="sortOption"
-        selected-class="white text-light-blue bg-white"
-        dark
-        mandatory
-        style="min-width: 25%"
+      <DandisetSearchField class="flex-grow-1 mr-2" />
+      <v-btn-group
+        variant="flat"
+        density="comfortable"
+        divided
+        class="btn-group--sort-options"
       >
-        <v-chip
-          v-for="(option, i) in sortingOptions"
-          :key="option.name"
-          @click="changeSort(i)"
+        <v-btn class="btn--sort-option">
+          <v-icon>mdi-sort-variant</v-icon>
```

Comment:
Should the "sort" icon also vary based on whether it's set to ascending/descending? I'm not sure myself, as it seems like the arrow icon currently serves that purpose. But for some reason it looks off to me

i.e. should we do this?
```suggestion
          <v-icon v-if="sortDir === 1">
            mdi-sort-variant
          </v-icon>
          <v-icon v-else>
            mdi-sort-reverse-variant
          </v-icon>
```

@jtomeck was this intended in the original design?

Reply:
601ac3d implements the design discussed earlier today, which swaps out the icon depending on the current sort order.